docs/MV3.md

```diff
@@ -104,11 +127,11 @@ This is a single rule that covers all the redirects from `ipfs.io` to `localhost
 
 Since the process is asynchronous, there are a few things that we need to keep in mind:
 
-- There will always be scenarios, where the redirection to public URL happens before companion determines if the URL is serviceable (consider ipns/fqdn resolutions.) To tackle this a refresh mechanism has been put in place that refreshes the page after a few seconds if the URL is serviceable. This is not ideal, but it is the best we can do for now.
+- There will always be scenarios, where the servicing of the public URL happens before companion determines if the URL is serviceable over IPFS (consider ipns/fqdn resolutions.) To tackle this a refresh mechanism has been put in place that refreshes the page after a few milliseconds if the URL is serviceable. This is not ideal, but it is the best we can do for now.
```

Comment:
Is there a way we can check if the page has fully rendered and *not* do the redirect if it has? i.e. if I've browsed to content and I am wanting to view it, *now*, then I don't want to wait to reload it again.

Also, do we have a tracking issue for discussing more optimal UX solutions for the "refresh after few-ms" funcitonality? I think a better experience would be companion prompting me that the page is available from my local node and allowing me to choose whether to refresh or not. Note: We may not need to solve this prior to mv3 launch, but we should document how to make it better and linking chrome tickets/questions to that need.